homeassistant/components/amcrest/binary_sensor.py

```diff
@@ -111,6 +115,12 @@ class AmcrestSensorEntityDescription(BinarySensorEntityDescription):
         device_class=BinarySensorDeviceClass.CONNECTIVITY,
         should_poll=True,
     ),
+    AmcrestSensorEntityDescription(
+        key=_DOORBELL_KEY,
+        name=_DOORBELL_NAME,
+        device_class=BinarySensorDeviceClass.OCCUPANCY,
+        event_code=_DOORBELL_EVENT_CODE,
+    ),
```

Comment:
This PR violated ADR-0007 and ADR-0010

- https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md
- https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

I suggest reverting this PR @Kane610 @PanicRide @flacjacket, as no longer allow integrations to add or change a platform YAML configuration.

Reply:
Dang it! So much for helping with the review effort, sorry... I'll revert it tonight

Reply:
Aww... so much for finding something simple for my first contribution.

Don't worry, I still love you @frenck! :)

Reply:
Well, it's not personal @PanicRide! I definitely am thankful for the willingness to contribute! ❤️

Reply:
After reading those ADR documents, I agree with the concepts.  I just wasn't aware of them.  I don't take it personally. :)

Reply:
New PR to revert published